portality/templates/dashboard/index.html

```diff
@@ -14,65 +14,72 @@
         constants.TODO_MANED_READY: {
             "text" : "<strong>Ready</strong> application",
             "colour" : "var(--dark-green)",
-            "feather": "thumbs-up"
+            "feather": "check-circle",
+            "status": "ready"
         },
         constants.TODO_MANED_COMPLETED: {
             "text" : "<strong>Completed</strong> application",
             "colour" : "var(--mid-green)",
-            "feather": "check-circle"
+            "feather": "user-check",
+            "status": "completed"
         },
         constants.TODO_MANED_ASSIGN_PENDING: {
             "text" : "<strong>Pending</strong> application",
             "colour" : "var(--yellow)",
-            "feather": "inbox"
+            "feather": "inbox",
+            "status": "pending"
         }
     }
 %}
 
 {% block content %}
-<main>
-  <h1>Hi, {{ current_user.name }}!</h1>
-
   <section>
-    <h2 class="type-04">Here are your priorities:</h2>
-    <ol class="unstyled-list todo-list">
+    <h2 class="type-04">
+      You have <strong>{{ todos|length }}{% if todos|length > config.get("TODO_LIST_SIZE")%}+{% endif %}</strong> items in your priority list:
```

Comment:
todo list size is never going to be bigger than the configured limit, so this test won't ever trigger.  I've made it a `>=` so that you get the `+` when there are 50